rfcs/InputUnion.md

```diff
@@ -288,11 +308,21 @@ type Mutation {
 }
 ```
 
-##### Variations:
+#### Variations:
 
 * A `default` type may be defined, for which specifying the `__typename` is not required. This enables a field to migration from an `Input` to an `Input Union`
 
-#### Single user-chosen field; value is the `type`
+#### Evaluation
+
+##### [Input unions should accept plain data from clients](#h-input-unions-should-accept-plain-data-from-clients)
+
+No, as there is an additional field that has to be added.
+
+##### [Input unions should be expressed efficiently in the query and on the wire](#k-input-unions-should-be-expressed-efficiently-in-the-query-and-on-the-wire)
+
+The additional field bloats the query.
```

Comment:
Are we really considering a single additional field to be "bloat"? The APIs I deal with already have hundreds of fields, I don't think 1 more used in some types makes much difference